code/common/pubsub.q

```diff
@@ -157,3 +157,41 @@ init:{[t]
   val:.u.sub[`$tab;1!enlist `tabname`filters`columns!(`$tab;filters;columns)];
   $[10h~type last val;'last val;val]
  };
+
+// Striping data in a TorQ Installation
+// Limitation: only max 16 (hexidecimal digits 0-9 A-F) processes using this logic
+// Can expand beyond 16 processes if required - Modify lookup and modmd5 function to use more digits e.g. "8b" for a single map
+.ds.lookup:{[numseg;maxproc]
+    hexdg:lower .Q.nA til maxproc;
+    seg:til numseg;
+    hexdg!maxproc#til numseg
+    }[;16]
+
+// Hash function
+.ds.modmd5:{first each string first each md5'[string x]}
+
+.ds.map:{[numseg;sym] sym@/:group .ds.lookup[numseg] .ds.modmd5 sym}
+
+// Get the number of rdb processes from process.csv
+if[.ds.numseg=0;.ds.numseg:sum `rdb=((.proc`readprocs).proc`file)`proctype]
```

Comment:
We shouldn't have any hardcoded requirements on process types or names. 

I'm not sure the best way to do this, but I don't think it needs to be full automatic as processes are added to the config file. I think it's fine to define in the publisher that the data is to be be split X ways, and that the subscriber has a config number that knows which consumer it is

Reply:
I have removed this in 70c9c8b833392c818eba63956cc1a024196d4226, users can specify the number in the extras column of the process.csv